bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/IntRSModule.java

```diff
@@ -17,19 +17,18 @@
 import java.io.OutputStream;
 import java.util.TooManyListenersException;
 
-import javax.naming.ConfigurationException;
-
+import javax.validation.constraints.NotNull;
```

Comment:
This is the wrong annotation, we use NonNull and its from another package :-)

Reply:
My mistake, done.